src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -0,0 +1,164 @@
+        contactsNewModifyPage.clickSaveContactButton();
+    }
+    /**
+     * Then step.
+     */
+    @Then("^The system shows the new contact$")
+    public void TheSystemShowsTheNewContact() {
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        contactsDetailPage = new SFCDetailsPage();
+        System.out.println("This Step tests new account creation:" + contactsDetailPage.newContactSavedName());
+        StringJoiner name = new StringJoiner(" ");
+        name.add(propertiesManager.getProperty("contactName"));
+        name.add(propertiesManager.getProperty("contactLastName"));
+        Assert.assertEquals(contactsDetailPage.newContactSavedName(), name.toString());
+    }
+    /**
+     * When step.
+     */
+    @When("^I choose a contact from recent contacts and I click on edit contact")
+    public void IChooseAContactFromRecentContactsAndIClickOnEditContact() {
+        System.out.println("Choose last contact");
+        contactsMainPage = new SFCMainPage();
+        contactsMainPage.clickContactNameLink();
+        contactsDetailPage = new SFCDetailsPage();
+        contactsDetailPage.clickEditContact();
+
+    }
+    /**
+     * And step.
+     */
+    @And("^I edit the contact name field and I press the save button")
+    public void IEditTheContactNameFieldAndIPressTheSaveButton() {
+        System.out.println("This Step edits contact name field");
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        contactsNewModifyPage = new SFCNewModifyPage();
+        contactsNewModifyPage.editContact(propertiesManager.getProperty("contactEditName"));
+        contactsNewModifyPage.clickSaveContactButton();
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^The system shows the modified contact$")
+    public void TheSystemShowsTheModifiedContact() {
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        contactsDetailPage = new SFCDetailsPage();
+        System.out.println("This Step tests modified contact:" + contactsDetailPage.newContactSavedName());
+        StringJoiner name = new StringJoiner(" ");
+        name.add(propertiesManager.getProperty("contactEditName"));
+        name.add(propertiesManager.getProperty("contactLastName"));
+        Assert.assertEquals(contactsDetailPage.newContactSavedName(), name.toString());
+    }
+    /**
+     * When step.
+     */
+    @When("^I choose a contact from recent contacts and I click on delete contact")
+    public void IChooseAContactFromRecentContactsAndIClickOnDeleteContact() {
+        System.out.println("Choose last contact");
+        contactsMainPage = new SFCMainPage();
+        contactsMainPage.clickContactNameLink();
+        contactsDetailPage = new SFCDetailsPage();
+        contactsDetailPage.clickDeleteContact();
+    }
+    /**
+     * And step.
+     */
+    @And("^I click on ok button")
+    public void IClickOnOkButton() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)